README.md

```diff
@@ -3,6 +3,12 @@
 This project contains the MLPerf mobile app, an app-based implementation of
 [MLPerf Inference](https://github.com/mlperf/inference) tasks.
 
+**TOC**:
```

Comment:
Maybe it would be better to spell it fully, without abbreviations?
And also have it be a proper header, with `##`, instead of just bold text?

Reply:
Yes. We can use the full text `Table of Contents`.
But bold text is better since TOC is not a section and does not have the same level of meaning like the other sections.